tests/sql/test_selectable.py

```diff
@@ -200,6 +200,94 @@ def test_nested_array_join(self):
             'ARRAY JOIN t1.parent AS p'
         )
 
+    def test_nested_left_array_join(self):
+        table = self._make_table(
+            't1',
+            Column('x', types.Int32, primary_key=True),
+            Column('parent', types.Nested(
+                Column('child1', types.Int32),
+                Column('child2', types.String),
+            ))
+        )
+        query = select(
+            [
+                table.c.parent.child1,
+                table.c.parent.child2,
+            ]
+        ).left_array_join(
```

Comment:
Why just not add keyword argument `left=True` to existing `array_join`? `select(...).array_join(..., left=True)`

Reply:
I thought about it, but decided to do it in a similar way, as in a regular join. `outerjoin` is a separate method. In fact, then I remembered that you can pass the `outer` argument to the `join` method. Therefore, in this case, indeed, it makes sense to add the `left` argument, which will be equivalent to calling the `left_array_join` method. How do you think?